docs/releases/v8_11.md

```diff
@@ -0,0 +1,35 @@
+# Release Notes
+
+## November 2022 (version 8.11)
+
+### Overview
+
+The **November 19th, 2022** release of **DietPi v8.11** comes with xxx and a couple of minor bug fixes.
+
+![Pumpkins](../assets/images/dietpi-release-v8_11.jpg){: width="480" height="320" loading="lazy"}
+
+!!! cite "*Photo by `AnnieSpratt`, Pixabay*"
+
+### New software
+
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | **Homer** :octicons-arrow-right-16: Added a simple `**HOM**epage` for your `serv**ER**` to keep your services on hand, with software ID 5. Many thanks to @t3dium for implementing this software title: <https://github.com/MichaIng/DietPi/pull/5833>
+
+### Improvements
+
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Shairport Sync**](../../software/media/#shairport-sync) :octicons-arrow-right-16: Updated to version 4.1. The update can be applied via reinstall: `dietpi-software reinstall 37`
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**NoMachine**](../../software/remote_desktop/#nomachine) :octicons-arrow-right-16: Bumped version to latest stable v8.1.2_1, update via reinstall: `dietpi-software reinstall 30`
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**HAProxy**](../../software/advanced_networking/#haproxy) :octicons-arrow-right-16: Bumped version to latest stable v2.6.6, update via reinstall: `dietpi-software reinstall 98`
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Grafana**](../../software/hardware_projects/#grafana) :octicons-arrow-right-16: For ARMv6 RPi models, RPi 1 and Zero (1), bumped version to latest stable v9.2.3, update via reinstall: `dietpi-software reinstall 77`
+
+### Bug fixes
+
+- [**DietPi-DDNS**](../../dietpi_tools/#dietpi-ddns) :octicons-arrow-right-16: Resolved an issue where IPv6 was forced when if was chosen to not force any IP family. Many thanks to @champymarty for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5830>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Navidrome**](../../software/media/#navidrome) :octicons-arrow-right-16: Resolved an issue where on reinstalls the config was was overwritten. Many thanks to @pedrom34 for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5840>
+- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Home Assistant**](../../software/home_automation/#home-assistant) :octicons-arrow-right-16: Resolved an issue where some modules didn't work on ARMv6 and ARMv7 systems because of missing runtime libraries. Many thanks to @mail2rst and @HyperCriSiS for reporting this issue: <https://dietpi.com/forum/t/after-the-update-homeassistant-mqtt-broker-integration-broken/14719>, <https://dietpi.com/forum/t/home-assistant-issues-after-latest-upate/14749>
```

Comment:
```suggestion
- [**DietPi-Software**](../../dietpi_tools/#dietpi-software) | [**Home Assistant**](../../software/home_automation/#home-assistant) :octicons-arrow-right-16: Resolved an issue where some modules didn't work on ARMv6 and ARMv7 systems because of missing runtime libraries. Many thanks to [mail2rst](https://dietpi.com/forum/u/mail2rst) and [HyperCriSiS](https://dietpi.com/forum/u/HyperCriSiS) for reporting this issue: <https://dietpi.com/forum/t/after-the-update-homeassistant-mqtt-broker-integration-broken/14719>, <https://dietpi.com/forum/t/home-assistant-issues-after-latest-upate/14749>
```